addons/hr_recruitment/wizard/applicant_send_mail.py

```diff
@@ -45,5 +56,24 @@ def action_send(self):
                 email_layout_xmlid='mail.mail_notification_light',
                 message_type='comment',
                 partner_ids=applicant.partner_id.ids,
-                subject=self.subject,
+                subject=subjects[applicant.id],
+                attachment_ids=self.attachment_ids.ids
```

Comment:
Seems like you'll have the same attachment for all applicants => Aka removing an attachment for 1 applicant will probably remove it for every applicants.

Reply:
If this will probably be never displayed in chatter as res_model/res_id is not transformed by _process_attachments_for_post (according to @tde-banana-odoo as well).